tests/testthat/test-box.R

```diff
@@ -565,59 +640,62 @@ test_that("box() returns appropriately formatted json", {
+#   dt_string <- box.dt(df, map_string)
   
-  map_num <- data.frame('id' = c('entity.cont1', 'entity.numcat2', 'entity.myoverlay'), 'plotRef' = c('yAxisVariable', 'xAxisVariable', 'overlayVariable'), 'dataType' = c('NUMBER', 'NUMBER', 'STRING'), 'dataShape' = c('CONTINUOUS', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
-  dt_num <- box.dt(df, map_num)
+#   map_num <- data.frame('id' = c('entity.cont1', 'entity.numcat2', 'entity.myoverlay'), 'plotRef' = c('yAxisVariable', 'xAxisVariable', 'overlayVariable'), 'dataType' = c('NUMBER', 'NUMBER', 'STRING'), 'dataShape' = c('CONTINUOUS', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
+#   dt_num <- box.dt(df, map_num)
   
-  expect_equal(nrow(dt_string), nrow(dt_num))
-  expect_equal(names(dt_string), names(dt_num))
-  expect_equal(lapply(dt_string, function(x) {length(x[[1]])}), lapply(dt_num, function(x) {length(x[[1]])}))
+#   expect_equal(nrow(dt_string), nrow(dt_num))
+#   expect_equal(names(dt_string), names(dt_num))
+#   expect_equal(lapply(dt_string, function(x) {length(x[[1]])}), lapply(dt_num, function(x) {length(x[[1]])}))
   
-  map_string <- data.frame('id' = c('entity.cont1', 'entity.strcat2', 'entity.myoverlay'), 'plotRef' = c('yAxisVariable', 'xAxisVariable', 'facetVariable1'), 'dataType' = c('NUMBER', 'STRING', 'STRING'), 'dataShape' = c('CONTINUOUS', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
-  dt_string <- box.dt(df, map_string)
+#   map_string <- data.frame('id' = c('entity.cont1', 'entity.strcat2', 'entity.myoverlay'), 'plotRef' = c('yAxisVariable', 'xAxisVariable', 'facetVariable1'), 'dataType' = c('NUMBER', 'STRING', 'STRING'), 'dataShape' = c('CONTINUOUS', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
+#   dt_string <- box.dt(df, map_string)
   
-  map_num <- data.frame('id' = c('entity.cont1', 'entity.numcat2', 'entity.myoverlay'), 'plotRef' = c('yAxisVariable', 'xAxisVariable', 'facetVariable1'), 'dataType' = c('NUMBER', 'NUMBER', 'STRING'), 'dataShape' = c('CONTINUOUS', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
-  dt_num <- box.dt(df, map_num)
+#   map_num <- data.frame('id' = c('entity.cont1', 'entity.numcat2', 'entity.myoverlay'), 'plotRef' = c('yAxisVariable', 'xAxisVariable', 'facetVariable1'), 'dataType' = c('NUMBER', 'NUMBER', 'STRING'), 'dataShape' = c('CONTINUOUS', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
+#   dt_num <- box.dt(df, map_num)
   
-  expect_equal(nrow(dt_string), nrow(dt_num))
-  expect_equal(names(dt_string), names(dt_num))
-  expect_equal(lapply(dt_string, function(x) {length(x[[1]])}), lapply(dt_num, function(x) {length(x[[1]])}))
+#   expect_equal(nrow(dt_string), nrow(dt_num))
+#   expect_equal(names(dt_string), names(dt_num))
+#   expect_equal(lapply(dt_string, function(x) {length(x[[1]])}), lapply(dt_num, function(x) {length(x[[1]])}))
   
-})
-
+# })
 
-test_that("box.dt() returns correct information about missing data", {
```

Comment:
did this move somewhere else and i didnt notice it?

Reply:
yup! just moved up to line 643